css/main.css

```diff
@@ -0,0 +1,494 @@
+  align-self: start;
+  transition: 350ms;
+}
+
+.link:hover {
+  background: #a04300;
+}
+
+.about-me-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.list-title {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages {
+  list-style: inside url("../media/svg/rectangle.svg");
+}
+
+.frameworks {
+  list-style: inside url("../media/svg/square.svg");
+}
+
+.skills {
+  list-style: inside url("../media/svg/circle.svg");
+}
+
+.contact-form-view {
+  height: 1000px;
+}
+
+form {
+  margin-top: -140px;
+  height: 400px;
+}
+
+input[type="email"] {
+  display: block;
+  box-sizing: border-box;
+  margin-bottom: 20px;
+  padding: 15px;
+  margin-left: -10px;
+  height: 52px;
+  border: none;
+  border-bottom: 1px solid #aaa;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  transition: 0.2s ease;
+}
+
+input[type="text"] {
+  display: block;
+  box-sizing: border-box;
+  margin-bottom: 20px;
+  margin-left: -10px;
+  padding: 15px;
+  height: 52px;
+  border: none;
+  border-bottom: 1px solid #aaa;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  transition: 0.2s ease;
+}
+
+/* input[type="submit"] {
+  margin-top: 68px;
+  width: 131px;
+  height: 52px;
+  background: #f55800;
+  border: none;
+  margin-left: 190px;
+  border-radius: 2px;
+  line-height: 24px;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  transition: 0.1s ease;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+} */
+
+.contact-section {
+  height: 1000px;
+  display: flex;
+  flex-direction: column;
+  gap: 58px;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-top: 100px;
+}
+
+.contact-text {
+  text-align: center;
+  position: static;
+  height: 216px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 30px;
+  padding-left: -10px;
+  line-height: 40px;
+  color: #172b4d;
+  margin: 24px 0;
+}
+
+.contact-form {
+  left: 24px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  position: static;
+  height: 330px;
+  top: 334px;
+  order: 1;
+  flex-grow: 0;
+  margin: 250px 24px;
+}
+
+#submit {
+  width: 150px;
+  text-align: center;
+  margin: 180px 100px;
+  padding: 12px;
+  border: 1px solid;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  color: #f4f5f7;
+  font-size: 17px;
+  background-color: #ff6b00;
+  line-height: 24px;
+}
+
+/* #submit:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+  box-sizing: border-box;
+} */
+
+#comment {
+  background: #fbf8f7;
+  position: absolute;
+  margin-left: -14px;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  line-height: 24px;
+  color: black;
+  margin-top: 16px;
+  height: 114px;
+  text-transform: capitalize;
+  border: none;
+  padding: 5px;
+}
```

Comment:
- [ ] You can improve this section by giving to the `textarea` the same width value you used for the input 👇🏻. This is to make sure that you follow the Figma design guideline and replicate the design to look almost identical to the original design. 

![kkkk](https://user-images.githubusercontent.com/79058364/152646288-1a5e2183-b5be-420f-8c9f-420bf4e670d5.png)